### ISSUE TITLE
Guides from README.md are down

### ISSUE THREAD

**Issue by JoooostB** [pmcg31/esp32_victron_test#1]
First things first, really like your work; thanks for sharing it with the world :). However, I tried to read the guide you linked in the  README.md, but it doesn't seem to exist anymore. Could you either commit it to the repo or make it available again?

Thanks!

**Comment by pmcg31** [comment]
Hello, and thank you for the kind words and bringing this issue to my attention. Somewhere along the line I rearranged some things and broke those links. They are now fixed, please give it another go and please let me know if there are further issues or if you find anything in the guides hard to follow. I want them to be as easy to read and follow as possible.

Just in case the links somehow still aren't working, the guides and a few other blog posts are on the site https://ideaup.online/.

--Rich
